src/openms/source/ANALYSIS/OPENSWATH/DIAScoring.cpp

```diff
@@ -201,17 +201,44 @@ namespace OpenMS
   }
 
   /// Precursor isotope scores
+  void DIAScoring::dia_ms1_isotope_scores(double precursor_mz, SpectrumPtrType spectrum,
+                                          double& isotope_corr, double& isotope_overlap, const EmpiricalFormula& sum_formula) const
+  {
+    //TODO precursor_mz can be received from the empirical formula (if required non-empty)
+
+    //collect the potential isotopes of this peak
+    int charge_state = sum_formula.getCharge();
```

Comment:
I probably have overlooked it, but where was the charge set previously. Is there a standard charge based on the polarity of the experiment in the charge was not set properly?

Reply:
I think the constructor of EmpiricalFormula takes a charge. Or AASequence.getFormula sets it automatically, too.